src/CommitmentsV2/SFA.DAS.CommitmentsV2/Extensions/QueryableExtensions.cs

```diff
@@ -0,0 +1,30 @@
+﻿using System.Linq;
+using System.Reflection;
+using Microsoft.EntityFrameworkCore.Query;
+using Microsoft.EntityFrameworkCore.Query.Internal;
+using Microsoft.EntityFrameworkCore.Storage;
+
+public static class QueryableExtensions
```

Comment:
this extension has no usages in the code.

Reply:
It is not used specifically, I have been using it in a watch - it's crucial see what SQL is actually created from the linq - as it contains configured auto predicates which are implicitly added